examples/dialogflow-intents-example/README.md

```diff
@@ -0,0 +1,23 @@
+# Intents creation and update for Dialogflow
+Intents for Dialogflow Agent can be created automatically from a CSV file using this module. The intents are created as a zip file which can then be manually imported in to the Dialogflow Project. 
+
+## Recomended Reading
+[Intents Overview](https://cloud.google.com/dialogflow/docs/intents-overview)
+
+## Programming Language
+Python 3
+
+## Project Structure
+```
+.
+├── BikeShopBotTrain.csv
+├── README.md
+├── bot_builder.py # program that builds the bot from the csv
+└── df_bot.py # Helper modules
+```
+
+## Usage
+python bot_builder.py --root_dir=bike_shop_bot_dir --bot_name=BikeShopBot < BikeShopBotTrain.csv
```

Comment:
Change the parameter name root_dir to output_dir since it's used for writing output
Add parameter called input_file to accept input file similar to other parameters instead of reading from stdin using bash style

e.g:  
```shell
python bot_builder.py \
--bot_name=BikeShopBot \
--input_file=BikeShopBotTrain.csv \
--output_dir=bike_shop_bot_dir
```